reports.php

```diff
@@ -111,7 +111,7 @@
 			}
 		}
 		if($count > 0 ){
-			echo '<label>Overview Reports: <select name="overview" class="select_report_dataset" class="input-xlarge" data-type="overview">'.$output.'</select></label>';
+			echo '<label>Overview Reports: <select name="overview" class="select_report_dataset" class="input-xlarge" data-type="overview" style="max-width: 90%; min-width: 90%; width: 90%">'.$output.'</select></label>';

```

Comment:
`width` overrides both `max-width` and `min-width`, so only `width: 90%` is required.